centml/sdk/api.py

```diff
@@ -44,25 +44,39 @@ def get_compute(id):
         return api.get_compute_deployment_deployments_compute_deployment_id_get(id)
 
 
-def create_inference(name, image, port, hw_id, health, min_replicas, max_replicas, username, password, env):
+def create_inference(**kwargs):
```

Comment:
yeah i understand but  the reason for tweeking the way the arguments are passed because it was giving lint warning on local, and may be it will fail the ci-check,  if we will give more argument to a function

Reply:
done